contributors/devel/README.md

```diff
@@ -49,10 +49,10 @@ Guide](http://kubernetes.io/docs/admin/).
 
 ## Developing against the Kubernetes API
 
-* The [REST API documentation](../api-reference/README.md) explains the REST
+* The [REST API documentation](http://kubernetes.io/docs/reference/) explains the REST
```

Comment:
I wasn't sure - I simply noticed the link was broken, so I tried to insert a generic pointer to the doc site that looked like it would accomodate information down the road. This stemmed from a question I asked in https://github.com/kubernetes/community/issues/235, looking for exactly this information - what's the current API and how to use it.